src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Cron\CronExpression;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
+
+    public static function getName(): string
+    {
+        return 'schedule-executions';
+    }
+
+    public static function getSupportedResource(): string
+    {
+        return 'execution';
+    }
+
+    protected function enqueueResources(Group $pools, Database $dbForConsole): void
+    {
+        foreach ($this->schedules as $schedule) {
+            if (!$schedule['active'] || CronExpression::isValidExpression($schedule['schedule'])) {
+                unset($this->schedules[$schedule['resourceId']]);
+                continue;
+            }
+
+            $now = new \DateTime();
+            $scheduledAt = new \DateTime($schedule['schedule']);
+
+            if ($scheduledAt > $now) {
+                continue;
+            }
+
+            \go(function () use ($schedule, $pools, $dbForConsole) {
+                $queue = $pools->get('queue')->pop();
+                $connection = $queue->getResource();
+
+                $queueForFunctions = new Func($connection);
+
+                $queueForFunctions
+                    ->setType('schedule')
+                    ->setFunctionId($schedule['resource']['functionId'])
+                    ->setExecution($schedule['resource'])
+                    ->setMethod('POST')
+                    ->setPath('/')
+                    ->setProject($schedule['project'])
+                    ->trigger();
```

Comment:
setType, setMethod and setPath doesnt sound right. Method and path should be coming from the execution itself - this worker isn't setting it.
setType might in theory stay, but behaviour is closer to HTTP execution, compared to scheduled one.

Reply:
Removed:
```
                    ->setMethod('POST')
                    ->setPath('/')
```

Do you have a suggestion for the alternative to `->setType('schedule')`?